PhotoBooth/SetupCardViewController.swift

```diff
@@ -0,0 +1,17 @@
+import UIKit
+
+class SetupCardViewController: UIViewController {
+
+    let setupCard = PhotoShootSetupCard()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureSetupCard()
+
+    }
+
+    private func configureSetupCard() {
+        view.addSubview(setupCard)
+        setupCard.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
Don't forget to set constraints on for the card. Right now you're using the frame size which is why it shows up correctly, but since you turned off `translatesAutoresizingMaskIntoConstraints` you have to use auto layout on the view itself.